lib/diagnostic.rb

```diff
@@ -27,6 +32,8 @@ def some_odd(numbers)
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
+    enum.each_with_object(true) { |o, m| m = false if o.odd? }
+    m
```

Comment:
See comments from previous question. The simpler way to do this would be to use `.all?`